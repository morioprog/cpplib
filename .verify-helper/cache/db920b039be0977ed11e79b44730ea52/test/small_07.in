8 20
62181893722 745086852306 415546960278 -505567026624 15731716888 -21986771895 -164261050284 972806574556
0 3 7 342478393304
1 0 7 -487119586333
3 0 4
3 4 6
2 2 5 -361264550486
2 1 6 -116332039666
1 0 6 -915781371359
0 2 4 977007853979
3 0 5
1 5 8 -747834035119
2 7 8 -66270242228
0 4 6 -550840467096
1 0 8 -228658293859
2 7 8 5986420488
0 0 2 759416666156
2 1 3 -753455452110
3 2 8
0 5 8 350581709706
2 0 8 453447496871
3 1 4
