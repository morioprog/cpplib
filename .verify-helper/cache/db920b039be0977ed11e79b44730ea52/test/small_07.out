735696119973
-6255055007
-748759168135
-753218261029
191478475300
